.github/pull_request_template.md

```diff
@@ -1,23 +1,79 @@
 <!--
+A few suggestions about filling out this PR
 
-Replace this comment with a description of what's being changed by this PR.
+1. Use a descriptive title for the PR.
+2. If this pull request is work in progress, create a draft PR instead of prefixing the title with WIP.
+3. Please label this PR at least one of the following labels, depending on the scope of your change:
```

Comment:
Use the exact label names here. I think we don't have labels for all of these yet.